playgrounds/app/src/components/Header.tsx

```diff
@@ -14,18 +15,65 @@ import {
 import { user } from '~/lib/store'
 import { TbCode, TbDoorExit } from 'solid-icons/tb'
 import { linkStyles } from '~/lib/styles'
+// import ModalComponent from './ui/modal'
+import ModalComponent1 from './modal'
+import { Dialog } from '@kobalte/core/dialog'
```

Comment:
We have the solid-ui version of this component in the ui folder. I would prefer to use that.